src/components/AssignedMembers/AssignedMembersList.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Row,
+  Col,
+  MultiColumnList,
+} from '@folio/stripes/components';
+
+import { COLUMN_MAPPING, COLUMN_WIDTH, VISIBLE_COLUMNS } from './constants';
+
+const AssignedMembersList = ({ users }) => {
+  const resultFormatter = {
```

Comment:
move it from component, it's non-changable and no reason to recreate is each render